### ISSUE TITLE
Drop mapbox dependency

### ISSUE THREAD

**Issue by tomalrussell** [nismod/irv-frontend#15]
Consider moving basemap to Deck.GL raster bitmap layer, drop react-map-gl and mapbox.

Downside - lose potential to have a vector basemap layer. Need to find an open satellite layer.

**Comment by tomalrussell** [comment]
@mz8i is this closed as of #28?

**Comment by mz8i** [comment]
Yes, although we're not dropping the dependency on w basemap library - quite the opposite, in fact - but the API token question is solved
